lib/src/system.dart

```diff
@@ -0,0 +1,45 @@
+import 'dart:io';
+
+import 'package:glob/glob.dart';
+import 'package:glob/list_local_fs.dart';
+import 'package:path/path.dart' as p;
+
+import 'collections.dart';
+import 'objects.dart';
+import 'strings.dart';
+
+// ignore: public_member_api_docs
+typedef FileSystemEntityPeekFunction = void Function(FileSystemEntity element);
```

Comment:
Why the `typedef`?
`void Function(FileSystemEntity element)` can be used in the function signature

Reply:
Doesn't really matter to dart. Inlined.